bin/fixmystreet.com/backfill-open311-comments

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env perl
+#
+# This script utilises the Open311 extension explained at
+# https://github.com/mysociety/FixMyStreet/wiki/Open311-FMS---Proposed-differences-to-Open311
+# to fetch updates on service requests for a specified timeframe, optionally for a single body
+
+use strict;
+use warnings;
+require 5.8.0;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use DateTime;
+use DateTime::Format::W3CDTF;
+use DateTime::Format::ISO8601;
+use Getopt::Long::Descriptive;
+
+my ($opt, $usage) = describe_options(
+    '%c %o',
+    [ 'body:s', "Name of body to use" ],
+    [ 'start_date:s', "Date of first comment to fetch" ],
+    [ 'end_date:s', "Date of last comment to fetch" ],
+    [ 'verbose', "Print out info as we go" ],
+    [ 'help', "print usage message and exit", { shortcircuit => 1 } ],
+);
+print($usage->text), exit if $opt->help;
+
+use Open311::GetServiceRequestUpdates;
+
+my $start_date = DateTime::Format::W3CDTF->parse_datetime( $opt->start_date );
+my $end_date = DateTime::Format::W3CDTF->parse_datetime( $opt->end_date );
+
+die "start and end date required\n" unless $start_date and $end_date;
+
+my $current_date = $start_date->clone;
+
+while ( $current_date < $end_date ) {
```

Comment:
Could do with a check that start is less than end, otherwise this will never end.